openshift-4.0/images/ose-etcd.yml

```diff
@@ -7,14 +7,18 @@ owners:
 
 content:
   source:
-    alias: ose-etcd
+    alias: etcd
     dockerfile: Dockerfile.rhel
 
-# from:
-#   builder:
-#   -
+from:
+  builder:
+  - image: openshift/golang-builder:1.10
```

Comment:
We also need a base image here. most likely:

```
  stream: rhel
```

Reply:
@sosiouxme thanks, I updated this I think it is what you are looking for.

Reply:
I said this without having looked at the Dockerfile, which I now see has origin-base as the base image. It's probably a good idea to make it "member: openshift-enterprise-base" instead of "stream: rhel", for consistency with both upstream and other images we ship. That said it should work either way.

Reply:
that makes sense to me updated